Fix problem where download_button CSS was ignored

ntoll opened the pull request:
# Description

Fixes #804.

Removes the line causing the hover state to be ignored. AFAICT the line that has been deleted shouldn't need to be there since the colour is correctly specified in the CSS for the FileWidget as is. I'm not sure why this was added. Can someone please check I'm not missing some wider context here..?

![hover](https://user-images.githubusercontent.com/37602/74429746-6d3a2280-4e53-11ea-90c9-c694aec0fba8.gif)


# Test Plan

Checked with Eyeball Mk.1 :eyes:

# Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:

 - [ ] I have tested these changes in the appropriate Qubes environment
 - [ ] I do not have an appropriate Qubes OS workstation set up (the reviewer will need to test these changes)
 - [X] These changes should not need testing in Qubes

If these changes add or remove files other than client code, packaging logic (e.g., the AppArmor profile) may need to be updated. Please check as applicable:

 - [ ] I have submitted a separate PR to the [packaging repo](https://github.com/freedomofpress/securedrop-debian-packaging)
 - [X] No update to the packaging logic (e.g., AppArmor profile) is required for these changes
 - [ ] I don't know and would appreciate guidance

ntoll commented:
@creviera I'm on my branch and it's rebased to current master, but I cannot reproduce the behaviour shown in your screenie (I added an `abort(409, "Bang")` to the `download_submission` method in the securedrop server so all file downloads fail). I'm so sorry about this, but do you have some STR then I'll fix things up. Thanks. :+1:

creviera commented:
To repro https://github.com/freedomofpress/securedrop-client/pull/805#pullrequestreview-360554293

you'll have to make it so the file fails to decrypt, you can use this STR: https://github.com/freedomofpress/securedrop-client/issues/754#issue-558416913

ntoll commented:
@creviera apologies... should have mentioned the force push fixes this. Rebased and ready for review from Friday.

creviera commented:
I'm still seeing this issue: https://github.com/freedomofpress/securedrop-client/pull/805#pullrequestreview-360554293. To repro see: https://github.com/freedomofpress/securedrop-client/pull/805#issuecomment-589352680

creviera commented:
i'll clean up my merge branch 'master' commits

ntoll commented:
@creviera huh..? That's *weird* because it is exactly the behaviour you say isn't fixed that I fixed (and checked) in my latest commit. Something's obviously gone wrong here -- I'll take a look and change as necessary. I'm so sorry for all this faffing about what should be a very simple fix. :-/

ntoll commented:
@creviera here's what I see:

![download_css](https://user-images.githubusercontent.com/37602/75765798-fa2c1980-5d37-11ea-8439-35f80cc7d070.gif)

The spinner resets to the correct colour and hover behaviour for me. Not quite sure what's going on. :-/

ntoll commented:
Ahhh... I think I found what you're referring to:

![download_css_fail](https://user-images.githubusercontent.com/37602/75765970-4a0ae080-5d38-11ea-96e5-3771bb9ef398.gif)

If the mouse is hovering over the download button when the state changes from "downloading" -> "download" (i.e. it's failed) then the icon doesn't change colour as it should. I'll try to fix.

ntoll commented:
Fixed, rebased and force pushed. @creviera I didn't catch this because the icon colour isn't controlled via CSS. Here's how it looks to me now:

![download_css_fixed](https://user-images.githubusercontent.com/37602/75767300-7f183280-5d3a-11ea-859d-2dafede3f365.gif)
